src/app/content/components/Content.tsx

```diff
@@ -159,30 +160,32 @@ const HideOverflowAndRedoPadding = isOpenConnector(styled.div`
  *   of things need to know when the sidebar is open/closed.
  */
 // tslint:disable-next-line:variable-name
-const Content: React.SFC = () => <Layout>
-  <GlobalStyle />
-  <Background>
-    <BookBanner/>
-    <Toolbar />
-    <Wrapper>
-      <CenteredContentRow>
-        <Sidebar />
-        <ContentPane>
-          <UndoPadding>
-            <MainContentWrapper>
-              <ContentNotifications />
-              <HideOverflowAndRedoPadding>
-                <Page />
-                <PrevNextBar />
-              </HideOverflowAndRedoPadding>
-              <Attribution />
-              <Footer/>
-            </MainContentWrapper>
-          </UndoPadding>
-        </ContentPane>
-      </CenteredContentRow>
-    </Wrapper>
-  </Background>
-</Layout>;
+const Content: React.SFC = () => <ErrorBoundary>
```

Comment:
putting this boundary here isn't functionally different from the one around the whole route, lets push it down around wrapper on 169 so they still see the navs if something in the content/toc/sidebar breaks